src/main/java/cafeboard/Board/Board.java

```diff
@@ -0,0 +1,43 @@
+package cafeboard.Board;
+
+import cafeboard.Post.Post;
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+public class Board {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String title;
+
+    @OneToMany(mappedBy = "board")
+    private List<Post> posts;
+
+    public Board() {
+    }
```

Comment:
사소한 거긴 한데, 개발자가 기본 생성자를 사용할 일이 없기 때문에 public보다 한 단계 접근 단계를 낮추는 편입니다.
```suggestion
    protected Board() {
    }
```